docker-compose.yaml

```diff
@@ -34,6 +34,8 @@ services:
       - /tmp/.X11-unix:/tmp/.X11-unix
       - ~/.Xauthority:/root/.Xauthority
     working_dir: "/project/TrustedComputeFramework/tools/build"
+    environment:
```

Comment:
Is this required, given that the setup.py, cmake/make files are assuming the default value to be SIM?

Reply:
@arsulegai. No this is not required. It makes it obvious for anyone using this script that it is for SGX simulator mode use only.